### DISCUSSION TITLE
possible typo in crate-status.md

### DISCUSSION THREAD

**Discussion by ToBinio** [GitoxideLabs/gitoxide#904] [General]
in the crate-status.md there is this line.

https://github.com/Byron/gitoxide/blob/5a9a7a31c1537aa54cd81a5145ca319a040b9951/crate-status.md?plain=1#L11

I believe instead of `git-*` it should be `gix-*`

or am I misunderstanding something?

**Comment by Byron** [comment]
Yes, that's correct, and a great catch. Would you like to be the one to fix it, too?

**Comment by ToBinio** [comment]
resolved in #905
